### DISCUSSION TITLE
Docker Compose file for Chandler

### DISCUSSION THREAD

**Discussion by b3nd1k** [monicahq/monica#7303] [Q&A]
Hey! I'm trying to get Chandler installed using docker compose, but I am running into issue. I would love it if someone can post a Docker Compose I can use to install Chandler with a Sqllite Database. Can anyone share it with me?

**Comment by despokd** [comment]
Look at https://github.com/monicahq/monica/issues/6626#issuecomment-1864246797

There are 2 examples in the issue comments, but a official compose template would be nicer :)

EDIT: replace `chandler` with `main` in the image tag.
EDIT2: https://github.com/monicahq/monica/discussions/7153 looks also promising
